test/scripts/e2e.sh

```diff
@@ -123,7 +123,11 @@ if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
     python3 -m venv "${TEMPDIR}/ve"
     . "${TEMPDIR}/ve/bin/activate"
     "${TEMPDIR}/ve/bin/pip3" install --upgrade pip
-    "${TEMPDIR}/ve/bin/pip3" install --upgrade py-algorand-sdk cryptography
+    "${TEMPDIR}/ve/bin/pip3" install --upgrade cryptograpy
+    # get the latest tip of our python SDK main branch as opposed to what's on pypi:
```

Comment:
I'm arguing that for testing in `go-algorand`, it's better to rely on the tip of our github than our latest official PyPi release. But honestly, in order to pull off my new `AtomicABI` Python test class, I needed this.